lib/ansible/modules/storage/netapp/na_ontap_qos_adaptive_policy_group.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/python
+
+# (c) 2018-2019, NetApp, Inc
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = '''
+module: na_ontap_qos_adaptive_policy_group
+short_description: NetApp ONTAP Adaptive Quality of Service policy group.
+extends_documentation_fragment:
+  - netapp.na_ontap
+version_added: '2.9'
+author: NetApp Professional Services - (@joshedmonds) <edmonds@netapp.com>
```

Comment:
Do you have a preference for field-contributed modules? "NetApp Professional Services" was just used to denote that it came from the field, but it doesn't have to be this. I can create an NG with all of 2 members in it, but not sure it's worth the overhead.

Reply:
should we use <ng-ansibleteam@netapp.com> like everything else?
It would be nice if you could join the ng-. 
Though our preferred support channel is slack.

Reply:
Sure. Requested an add to the NG. I'm also @joshedmonds in the netapp.io slack.

Reply:
Updated to match the standard format.